Spring.jsx

```diff
@@ -386,7 +303,7 @@ export let TransitionSpring = React.createClass({
       let mergedVals = mergeDiffObj(
         currVals,
         endValue,
-        key => willLeave(key, endValue, currVals, currV),
+        key => willLeave(key, endValue, currVals, currV)
```

Comment:
I'll let this be but I might put it back another time for making arguments easier to move and around and diff. Babel takes this and removes it anyways for backward compat